chapters/qa-4.mdx

```diff
@@ -200,11 +200,11 @@ let osoba = {
 }
 ```
 
-Pokud chceme zjistit věk naší osoby, musíme to udělat přes tečkovou notaci a napsat osoba.age. Pokud chceme zjistit jestli je osoba plnoletá, použijeme také tečkovou notaci, ale protože je to funkce přidáme závorky (musíme funkci zavolat - to platí u jakékoliv funkce, nikoliv těch, které patří do nějakého objektu) => osoba.isAdult().
+Pokud chceme zjistit věk naší osoby, musíme to udělat přes tečkovou notaci a napsat osoba.age. Pokud chceme zjistit, jestli je osoba plnoletá, použijeme také tečkovou notaci, ale protože je to funkce, přidáme závorky (musíme funkci zavolat - to platí u jakékoliv funkce, nikoliv těch, které patří do nějakého objektu) => osoba.isAdult().
```

Comment:
I do not understand the meaning of the text in ()

Reply:
Improved.